evalml/pipelines/component_graph.py

```diff
@@ -356,10 +378,10 @@ def __iter__(self):
         return self
 
     def __next__(self):
-        """Iterator for graphs, prints the components in the graph in order
```

Comment:
I don't think it is printing..? So updating docstring, lmk if I missed anything 🤔